cores/migrations/20240708182646_AddCategoryMarkdown.down.sql

```diff
@@ -0,0 +1,2 @@
+-- Add migration script here
+DROP TABLE IF EXISTS CategoryMarkdown;
```

Comment:
Wonder if we also need/want to drop the `Category.category_id` column in this `down`, since it got added in the `up` right?